notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/service/jpa/JpaEntryPostProcessor.java

```diff
@@ -0,0 +1,40 @@
+package org.jasig.portlet.notice.service.jpa;
+
+import org.jasig.portlet.notice.rest.EntryDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Mapping to fix the attribute/entry relationship after the attributes have been converted.
+ * Since the DTO objects don't have the link back to the entry, this doesn't get set.  Could
+ * make dozer do the addAttribute() call instead of setting the list directly, but that
+ * requires that the class no longer be package private.
```

Comment:
I appreciate the JavaDoc on this class.  I'd like to see all the classes in this changeset have a spiritually similar level of JavaDoc.